tools/cldr-apps/src/main/webapp/cldr-setup.jsp

```diff
@@ -2,8 +2,6 @@
 <%@page import="java.sql.SQLException"%>
 <%@page import="org.unicode.cldr.util.VoteResolver.VoterInfo"%>
 <%@page import="com.ibm.icu.util.VersionInfo"%>
-<%@page
-	import="org.tmatesoft.sqljet.core.internal.lang.SqlParser.commit_stmt_return"%>
```

Comment:
How does SqlParser relate to this ticket?

Reply:
- the import should not have been in this jsp 
- this PR removes the tmatesoft jars including the one imported here.